src/main/java/frc/robot/commands/defaultCommands/IntakeDefaultCommand.java

```diff
@@ -0,0 +1,31 @@
+package frc.robot.commands.defaultCommands;
+
+import java.util.function.DoubleSupplier;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.IntakeSubsystem;
+
+public class IntakeDefaultCommand extends Command {
+
+    private final IntakeSubsystem intake;
+    private final DoubleSupplier intakeSupplier;
+
+    public IntakeDefaultCommand(IntakeSubsystem intake, DoubleSupplier intakeSupplier) {
+        this.intakeSupplier = intakeSupplier;
+        this.intake = intake;
+        addRequirements(intake);
+
+        //TODO:Remove after command has been tested
+        throw new UnsupportedOperationException("IntakeDefaultCommand has not been tested on robot yet");
+    }
+
+    @Override
+    public void execute() {
+        intake.setIntakeSpeed(intakeSupplier.getAsDouble());
+    }
+
+    public void end() {
+        intake.setIntakeSpeed(0);
+    }
+    
```

Comment:
this doesn't actually override end. put a boolean interrupted parameter in there, then add an @override above, like
@Override
public void end(boolean interrupted)

Reply:
Whoa, great catch @oliverstivers. I missed that one!